CameraOverlay/configs.json

```diff
@@ -0,0 +1,14 @@
+{
+  "activeOverlay": "Overlay_Demo2.py",
+  "device": "primary",
+  "overlays": [
+    {
+      "file": "Overlay_Demo.py",
+      "name": "My first overlay"
```

Comment:
These will need more descriptive names, but I have no clue what each overlay actually looks like, so may have to wait on this

Reply:
We can change that later!